### PR TITLE
Add Github Actions workflow

### PR DESCRIPTION
**Pull request by paracycle**

### CODE REVIEW

**Comment by KaanOzkan** [comment]
Do we want to replace travis in this PR or in a separate PR?

File: Gemfile.lock [hunk 1]
```diff
@@ -107,7 +107,7 @@ GEM
     mimemagic (0.3.5)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
-    minitest (5.14.1)
+    minitest (5.14.2)
```
Comment (KaanOzkan) [inline Gemfile.lock:110]: Not really important but this is left over from the now scrapped "try to get Ruby Head working" work

**Comment by rafaelfranca** [comment]
> Do we want to replace travis in this PR or in a separate PR?

I'd do in the same PR.

**Review by rafaelfranca — APPROVED** [review]

**Comment by KaanOzkan** [comment]
image
 Does this come from enabling travis on this repo? Do we need to remove this repo from Travis UI? I don't want to sign in to Travis through github blindly, it's asking for permission to read organizations teams and private project boards.

**Comment by rafaelfranca** [comment]
This come from the branch protection config. I just updated the config to not require travis anymore and instead require github actions.

**Pull request merged**